README.md

```diff
@@ -1,31 +1,403 @@
-# Scaffolding
+# What is Scalprum? 
 
-## Introduction
+## “Portals on steroids!”
```

Comment:
@karelhala Do we want to mention portals? I have two comments.
1. Many developers might know even know what they are.
2. Those who know might not be that enthusiastic about its "return".

Maybe mention something about easy way to develop micro frontends or something like that.

Reply:
I removed this and can add when we are more confident about our byline.